src/components/App.js

```diff
@@ -18,49 +18,30 @@ const styles = {
   },
 };
 
-class App extends Component {
-  state = {
-    text: 'Venture first mover advantage learning curve market ecosystem funding stealth disruptive social proof scrum project growth hacking niche market user experience graphical user interface.',
-    entities: [
-      { start: 160, end: 184, label: 'very important'},
-      { start: 144, end: 159, label: 'very important'},
-      { start: 62, end: 69, label: 'important' },
-      { start: 116, end: 130, label: 'nonsense' },
-      { start: 8, end: 29, label: 'nonsense' },
-    ],
-  }
```

Comment:
This state is unnecessary at the App level, as it is consumed and updated in the `EntityHighlighter` component. Having it here also generates unneccessary re-renders of `App` when the state is updated. Ideal case for "collocation" of state, down to the child component `EntityHighlighter`.